src/app/dashboard/mycommunity/page.tsx

```diff
@@ -1,50 +1,66 @@
 'use client';
 
-import Image from 'next/image';
-import Link from 'next/link';
-import { Roboto } from 'next/font/google';
-import { Mixpanel } from '@/components/Mixpanel';
+import { useState, useEffect } from 'react';
 
-const roboto = Roboto({
-  weight: '300',
-  subsets: ['latin'],
-});
+import IntroCommunity from '../../../components/community/IntroCommunity';
+import CreateCommunity from '../../../components/community/CreateCommunity';
+import JoinCommunity from '../../../components/community/JoinCommunity';
+import CommunitySection from '../../../components/community/CommunitySection';
 
 export default function MyCommunity() {
+  const [introModal, showIntroModal] = useState(true);
```

Comment:
Please check how the other modal works like Create Collection, Mint NFT, Profile